src/objectid.ts

```diff
@@ -72,7 +99,7 @@ export class ObjectId extends BSONValue {
    *
    * @param inputId - A 12 byte binary Buffer.
    */
```

Comment:
This isn't a constructor we need to add to our public API, if you stick `@internal` on it then it gets deleted from the public Typescript :) Also, can you add the inputIndex `@param` docs?